blocks/feed/feed.js

```diff
@@ -0,0 +1,65 @@
+import {
+  buildBlock, createOptimizedPicture, decorateBlock, loadBlock, readBlockConfig, getMetadata,
+} from '../../scripts/lib-franklin.js';
+import { queryIndex } from '../../scripts/scripts.js';
+
+const resultParsers = {
```

Comment:
What is the resultParsers for? Maybe add a comment to explain a little...

Reply:
> What is the resultParsers for? Maybe add a comment to explain a little...

Good idea. Done.